xml/System.Web/IHttpHandler.xml

```diff
@@ -92,7 +92,7 @@
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- Place your custom `HttpHandler` code in the <xref:System.Web.IHttpHandler.ProcessRequest%2A> virtual method, as shown in the following example.  
+ Place your custom `HttpHandler` code in the <xref:System.Web.IHttpHandler.ProcessRequest%2A> virtual method, as shown in the previous example.  
```

Comment:
```suggestion
 Place your custom `HttpHandler` code in the <xref:System.Web.IHttpHandler.ProcessRequest%2A> virtual method, as shown in the [Examples](#examples) section.  
```

Reply:
Please forgive me for not responding more promptly. @rpetrusha's comment was excellent and I quite agree.